pkg/command/cmd_executor_test.go

```diff
@@ -203,7 +202,7 @@ var _ = Describe("Command executor features", func() {
 				Expect(executed).To(BeFalse())
 			})
 
-			It("should execute command because all conditions are fulfilled so no gock-not-mathing-error is expected ", func() {
+			It("should execute command because all conditions are fulfilled so no gock-not-matching-error is expected ", func() {
```

Comment:
Not sure why we refer to Gock in the feature description, just fixed typos here :)